ratis-examples/src/main/bin/client.sh

```diff
@@ -32,4 +32,4 @@ shift
 subcommand="$1"
 shift
 
-java ${LOGGER_OPTS} -jar $ARTIFACT "$example" "$subcommand" $QUORUM_OPTS "$@"
```

Comment:
Alternatively we can keep `QUORUM_OPTS` and then have a default setup for client.sh/server.sh so people can start client/server without adding the peers.

But that will be overlapped with the purpose of `start-all.sh` so I think remove `QUORUM_OPTS` is ok.

Reply:
Thank you, I also think it’s better to let users set their own peers parameters